tests/lib/rules/multiline-ternary.js

```diff
@@ -0,0 +1,276 @@
+/**
+ * @author dev1437
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+const { RuleTester, ESLint } = require('../../eslint-compat')
+const rule = require('../../../lib/rules/multiline-ternary')
+const semver = require('semver')
+
+const tester = new RuleTester({
+  parser: require.resolve('vue-eslint-parser'),
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module'
+  }
+})
+
+tester.run('multiline-ternary', rule, {
+  valid: [
+    {
+      filename: 'test.vue',
+      code: `
+      <template>
+        <div :class="{
+          'test': someReallyLongCondition ?
+            aVeryLongOutput :
+            thisCantFitOnASingleLine
+        }">
+        </div>
+      </template>
+      `
+    },
+    {
+      filename: 'test.vue',
+      code: `
+      <template>
+        <div :class="{
+          'test': someReallyLongCondition ? aVeryLongOutput : thisCantFitOnASingleLine
+        }">
+        </div>
+      </template>
+      `,
+      options: ['never']
+    }
+  ],
+  invalid: [
+    ...(semver.gte(ESLint.version, '7.1.0')
```

Comment:
I think it is better to branch at the output property instead of branching here.

```js
     output: semver.gte(ESLint.version, '7.0.0')
       ? ` .... `
       : null,
```